content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
+-> **Note:** As of September 2021, Run Tasks are a private beta feature, subject to change, and not all customers will see this functionality in their Terraform Cloud organization.
 
 
-Run Tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. The beta release of this feature allows users to add and execute these tasks during the new pre-apply stage which exists in between the plan and apply stages. Tasks are executed by sending an API payload to the external system. This payload contains a collection of run-related information and a callback URL which the external system can use to send updates back to Terraform Cloud.
+Run Tasks allow you to directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. When executed, a run task will send an API payload to the external service. This payload contains a collection of run-related information, including a callback URL which the service uses to send run task data back to Terraform Cloud.
 
-The external system can then use this run information and respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the task's enforcement settings within a workspace.
+The external service will use this run information to respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings that have been configured for the workspace.
 
 -> **API:** See the [Run Tasks APIs](../api/run-tasks.html).
 
 ## Requirements
 
-Run tasks can only be created on workspaces using a Terraform version v0.12+. Downgrading a workspace with existing run tasks to use a Terraform version v0.12 or prior will not result in an error, but the configured tasks will not execute.
+**Terraform Versions** - Run tasks can only be assigned to workspaces that use a Terraform version of 0.12 and later. Downgrading a workspace with existing run tasks to use a prior Terraform version will not result in an error, but the configured tasks will not execute as expected.
 
-You must be an organization owner to create a new task event hook. You must be at least a workspace administrator in order to connect a task event hook to a given workspace.
+**Permissions** - In order to create a run task, you must have a user account that has been assigned the organization owner permission. You must be at least a workspace administrator in order to associate a run task to a given workspace.
 
 ## Configuring a Run Task
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu. A page appears where you can either create a new event hook or select an existing one that is not connected to the workspace.
+1. Navigate to the desired workspace, then Click **Run Tasks** from the **Settings** menu.
```

Comment:
```suggestion
1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
```

Should `click` be lowercase?

Reply:
TBH, I am not sure but I do see that we use lower in the SAML docs.